dipy/denoise/localpca.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import scipy as sp
+
+def localpca(arr, sigma, patch_radius=1, tou=0, rician=True):
+    '''
+    Local PCA Based Denoising of Diffusion Datasets
+
+    References
+    ----------
+    Diffusion Weighted Image Denoising Using Overcomplete Local PCA
+    Manjon JV, Coupe P, Concha L, Buades A, Collins DL
+
```

Comment:
Simplify first line description of function - add a reference instead of list of names: "Diffusion Weighted Image Denoising Using Overcomplete Local PCA [1]_"